onboarding/chat-moderators.md

```diff
@@ -13,36 +15,65 @@ Removal of the role is subject to the discretion of the Core team members and ma
+        - Some examples include having deeper discussion regarding any continuing problematic behavior of theirs, to request the user change their inappropriate profile picture, or to inform a user of the evolved and problematic/toxic meaning of popular phrases/memes they might be using.
+    - If you need to speak to several people at once, such as when multiple people are dogpiling or having an off topic conversation, you should send a blanket message reminding users of the rules or asking users to stop (without calling anyone out by name).
+        - For example, "This sounds like an interesting discussion, but let's take it to `#odin-offtopic` instead."
+        - In cases where multiple users are dogpiling, it can be helpful to afterwards create a separate private thread for each user that was dogpiling to explain why their interaction was considered dogpiling.
+- **Temporarily mute users to prevent situations from blowing up**
+    - Use Dyno's `mute` command to temporarily mute a user, being sure to include a time limit and a reason in the command.
+        - A couple of examples for when you should temporarily mute a user is if they are getting visibily frustrated and taking it out on other users, or there is some sort of conflict occurring and you need to shut it down immediately before it gets worse.
+- **Give users the `NOBOT` role for spamming odin-bot commands**
+    - Users with this role will result in `odin-bot` ignoring any bot commands made by that user.
+        - A couple of examples of what constitutes spamming of the bot commands include giving a user multiple points in the span of a few minutes, or running random commands in random channels after being told to use the `#bot-spam-playground` channel. 
+- **Give users the `loki?` role for constant rule-breaking and "stern" warnings**
+    - The `loki?` role removes a user's permissions such that they can only view the `#please-wait`, `#please-wait-2`, and `#rules` channels.
+    - We apply this role to users who we wish to have a private conversation with, as the next step up from private threads, so that they cannot interact with the rest of the server and potentially continue breaking the rules.
+    - The `#please-wait` and `#please-wait-2` channels are public to other members of the team for transparency.
+    - `#please-wait-2` should be used moving forward in order to have engagements regarding the rules or general chat conduct. When `loki?`ing a user, you must create a private thread inside this channel like you would any other channel.
+    - Be mindful that the chat history is not preserved from the user perspective. If a user goes offline, when they return the chat history will be removed from their perspective, potentially requiring us to screenshot prior history.
```

Comment:
The chat history not being preserved should only be true for PW1, part of the push for PW2 was to get around that problem.

Reply:
Is it confirmed that chat history is preserved in PW2? I believe you and Leo had tested the channel, but I can't remember seeing if it was confirmed the history was preserved or not.